index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Adaobi Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+    <nav class="container">
+      <a href="#" class="navbar-logo">Adaobi Okoye</a>
+      <a href="#" class="navbar-menu">
+        <img src="/Icons/Union.png" alt="Menu"/>
+      </a>
+    </nav>
+    </header>
+    <section class="header-box">
+      <section class="header-content">
+        <h1 class="header-title">Hey There. I'm Adaobi</h1>
+        <h2 class="header-subtitle">I’m a Software Developer</h2>
+        <p class="header-text-description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/Icons/github.png" alt="Github"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="Icons/linkedin.png" alt="Linkedin"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/icons/angel.png" alt="Angel"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/Icons/twitter.png" alt="Twitter"/>
+            </a>
+          </li>
+          <li>
+            <a href="#"  class="social-media-medium-icon">
+              <img class="image-correction" src="/icons/medium.png" alt="Medium"/>
+            </a>
+          </li>
+        </ul>

```

Comment:
kindly change the location of your files. instead of `/Icons/Union.png"` use `icons/..."` followed by the name of tour icon and it will work